config.env

```diff
@@ -13,5 +11,6 @@ BN_CONSENSUS_SNAPSHOTS_PERIOD=240
 BN_CONSENSUS_APPLY_URL=http://localhost:8529/_db/_system/apply{v}/operations/{hash}
 BN_CONSENSUS_DUMP_URL=http://localhost:8529/_api/replication/dump
 BN_CONSENSUS_MAX_COLLECTION_SIZE=1000000000
-BN_CONSENSUS_SEED_VOTING_ADDRESS=0x56741DbC203648983c359A48aaf68f25f5550B6a
-BN_SP_UPDATER_SP_ADDRESS=0x0aB346a16ceA1B1363b20430C414eAB7bC179324
+BN_UPDATER_SEED_VOTING_ADDRESS=0x56741DbC203648983c359A48aaf68f25f5550B6a
+BN_UPDATER_SP_ADDRESS=0x0aB346a16ceA1B1363b20430C414eAB7bC179324
+BN_UPDATER_SP_INFURA_URL=wss://mainnet.infura.io/ws/v3/36e48f8228ad42a297049cabc1101324
```

Comment:
Who does this key belong to? If it's checked in, can't other people use up the quota.

Reply:
This was removed before and mistakenly returned while renaming SP_UPDATER to UPDATER
I removed the value again to ask node operators config it themselves with their own keys